File: MyMindNotes/app/src/main/java/com/android/mymindnotes/model/retrofit/ChangePasswordApi.java
package com.android.mymindnotes.model.retrofit;

import com.android.mymindnotes.model.ChangeUserPassword;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.PUT;

public interface ChangePasswordApi {
    @PUT("/api/member/update/password")
    Call<Map<String, Object>> updateUserPassword(@Body ChangeUserPassword changeUserPassword);
}

File: MyMindNotes/app/src/main/java/com/android/mymindnotes/data/retrofit/api/user/ChangePasswordApi.kt
package com.android.mymindnotes.data.retrofit.api.user

import retrofit2.http.PUT
import com.android.mymindnotes.data.retrofit.model.ChangeUserPassword
import retrofit2.http.Body

interface ChangePasswordApi {
    @PUT("/api/member/update/password")
    suspend fun updateUserPassword(
        @Body changeUserPassword: ChangeUserPassword
    ): Map<String, Object>
}